scala/src/main/scala/smile/validation/MutualInformationScore.scala

```diff
@@ -33,7 +34,7 @@ import scala.math.{max, log, sqrt}
    *
 	 * @author Beck Gaël
    */
-class MutualInformationScore(normalization: String = "sqrt") extends ClusterMeasure {
+class MutualInformationScore(normalization: Normalization = SQRT) extends ClusterMeasure {
```

Comment:
Please don't make this change. I understand your intent from Java/Scala point of view. But I try to provide an interface here similar to R/Matlab.